be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -76,6 +85,27 @@ class SegmentIterator::BitmapRangeIterator {
         return true;
     }
 
+    bool get_next_range_without_bitmap_index(uint32_t max_range_size, uint32_t* from, uint32_t* to) {
+        if (_curr_pos == _max_value) {
+            return false;
+        }
+        if (_curr_pos + max_range_size <= _max_value) {
```

Comment:
should we add `_eof` like `get_next_range_with_bitmap_index()`? ( i am not sure)

Reply:
```_eof``` is used to confirm whether reachs the end of ```roaring_read_uint32_iterator```.
If you don't use ```roaring_read_uint32_iterator```, then ```_eof``` is useless.

Reply:
i get it